CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.61] - 2022-10-04
```

Comment:
Similarly update package_version in wheel action? https://github.com/microsoft/DeepGNN/blob/7b92623cdd31404b5996390fb8597c8efbbb056d/.github/actions/wheel/action.yml#L11

Reply:
wheel action takes version as an argument. What exactly needs to be updated?

Reply:
I see, but should the default at least be 9.9.99 or something so it is not confused with a real version?

Reply:
default is `0.1.1`. Which is smaller than any existing package, in case wheel is published accidentally.